examples/light-switch-app/silabs/src/LightSwitchMgr.cpp

```diff
@@ -159,3 +286,80 @@ void LightSwitchMgr::GenericSwitchWorkerFunction(intptr_t context)
 
     Platform::Delete(data);
 }
+
+void LightSwitchMgr::ButtonEventHandler(uint8_t button, uint8_t btnAction)
```

Comment:
Can you provide information on the logic of the function?

Reply:
Added the brief in LightSwitchMgr.h file